src/app/myworkflows/my-workflow/my-workflow.component.ts

```diff
@@ -210,6 +212,16 @@ export class MyWorkflowComponent extends MyEntry implements OnInit {
     this.myWorkflowsService.registerEntry(this.entryType);
   }
 
+  addToExistingWorkflows(): void {
+    this.userQuery.user$
```

Comment:
don't subscribe, if the user in the state changes (for w/e reason), then you'd have two addUserToWorkflows calls.  

Just do userQuery.user$.getValue() to get the current user.  disable button when user is falsey.